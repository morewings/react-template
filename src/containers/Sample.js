import Sample from 'components/Sample';

// redux
import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';
import { sample } from 'Redux/actions/index';

function mapStateToProps(state, props) {
  return {
    increment: state.sample.increment,
  };
}

function mapDispatchToProps(dispatch) {
  return bindActionCreators({ sample }, dispatch);
}

export default connect(mapStateToProps, mapDispatchToProps)(Sample);
