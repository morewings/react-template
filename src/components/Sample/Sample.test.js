import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Sample from 'components/Sample/Sample';

describe('src > components > Sample.js', () => {
  const wrapper = shallow(<Sample />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
