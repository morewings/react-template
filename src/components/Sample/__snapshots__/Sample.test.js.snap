// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div>
    Hello from sample component!
  </div>
  <button>
    Click me
  </button>
  <div>
    You clicked: 
     times.
  </div>
</div>
`;
